evalit/odata/odata_automation.py

```diff
@@ -1,9 +1,8 @@
 class OdataAutomation:
-
     def __init__(self, config_yaml) -> None:
         with open(config_yaml) as f:
             self.config = yaml.load(f)
 
     def run_automation(self):
         # Implementation here
-        return [[0, 10], [2, 30], [0, 50]]
\ No newline at end of file
+        return [[0, 10], [2, 30], [0, 50]]
```

Comment:
some comments or variable names on why these numbers are this way would help someone taking a quick look at the code

Reply:
I have removed these dummies. Thanks for pointing this out.